### PR TITLE
Fix 403/404 errors from mapbox-gl trying to access levels that don't exist

### PR DESCRIPTION
**Pull request by shadc**
Set the maxzoom in the source style to be that of the vector service.

### CODE REVIEW

**Comment by jgravois** [comment]
very cool!  i'll try and find time to test this today.

**Comment by shadc** [comment]
Tired last night... did not test very thoroughly.  Needs a quick fix for services with metadata.maxzoom of undefined.

File: src/Util.js [hunk 1]
```diff
@@ -49,7 +49,8 @@ export function formatStyle (style, metadata, styleUrl) {
       style.sources.esri.url + metadata.tiles[0]
     ],
     description: metadata.description,
-    name: metadata.name
+    name: metadata.name,
+    maxzoom: metadata.maxzoom ? metadata.maxzoom : 22
```
Comment (jgravois) [inline src/Util.js]: might make more sense to use the same value as our hosted vector tiles as the default.

https://basemaps.arcgis.com/v1/arcgis/rest/services/World_Basemap/VectorTileServer?f=json

```js
"lods": [
  // ...
  {
    "level": 18,
    "resolution": 0.29858214164761665,
    "scale": 1128.497176
  }
]
```

**Comment by shadc** [comment]
Ok, changed it to 18.

**Comment by jgravois** [comment]
awesome. thanks again.

**Pull request merged**